src/views/self/view_teleport.cpp

```diff
@@ -109,28 +109,48 @@ namespace big
 
 		const auto& selected_ipl = ipls[g.self.ipls.select];
 		if (components::button("LOAD_IPL"_T.data()))
-		{
-			//unload all previous ipls
-			for (auto& ipl : ipls)
-				for (auto& ipl_name : ipl.ipl_names)
-				{
-					if (STREAMING::IS_IPL_ACTIVE(ipl_name))
+		{	
```

Comment:
components::button has a method of invoking queuing a fiber without explicitly calling queue_job. See this example here:  https://github.com/YimMenu/YimMenu/blob/d6c32533be18aef5b35122b4f068b8844155751c/src/views/self/view_weapons.cpp#L249-L251

Reply:
Also, components::button takes a string_view as a parameter, there is no need to invoke the .data() accessor on it to convert it to a const char*, which will then be re-cast to a string_view. You're basically doing string_view (translation) -> const char* (.data()) -> string_view (parameter) -> const char* (components::button's invoke of ImGui)